Userland/Libraries/LibAudio/FlacLoader.cpp

```diff
@@ -410,12 +398,15 @@ MaybeLoaderError FlacLoaderPlugin::next_frame(Span<Sample> target_vector)
     [[maybe_unused]] u16 footer_checksum = LOADER_TRY(bit_stream.read_bits<u16>(16));
     dbgln_if(AFLACLOADER_DEBUG, "Subframe footer checksum: {}", footer_checksum);
 
-    Vector<i32> left;
-    Vector<i32> right;
+    float sample_rescale = static_cast<float>(1 << (pcm_bits_per_sample(m_current_frame->bit_depth) - 1));
```

Comment:
Note: we can likely use an inverse here
...Than it will be a factor, as the debug message suggests,
and it may be measurably faster